### PR TITLE
refactor(rust): `project-member` commands, and adds the `show` command

### PR DESCRIPTION
**Pull request by adrianbenavides**

### CODE REVIEW

File: implementations/rust/ockam/ockam_command/src/project_member/delete.rs [hunk 1]
```diff
@@ -28,78 +30,147 @@ pub struct DeleteCommand {
...
+
+#[derive(Serialize)]
+struct DeleteMemberOutput {
+    project: String,
+    identifiers: Vec<Identifier>,
+}
+
+impl Display for DeleteMemberOutput {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        match self.identifiers.len() {
```
Comment (etorreborre) [inline implementations/rust/ockam/ockam_command/src/project_member/delete.rs]: Pattern matching on the slice would be a bit more elegant:
```
match self.identifiers[..] {
    [] => "There are no members",
    [identifier] => "Identifier {} ...",
    _ => "n members were deleted", 
}
```

File: implementations/rust/ockam/ockam_command/src/project_member/list.rs [hunk 2]
```diff
@@ -24,55 +23,49 @@ pub struct ListCommand {
...
         let members = authority_node_client
             .list_members(ctx)
             .await?
             .into_iter()
-            .map(|i| MemberOutput(i.0, i.1))
-            .collect();
-
-        print_members(&opts, members)?;
+            .filter(|(_, a)| {
+                if self.enrollers {
```
Comment (etorreborre) [inline implementations/rust/ockam/ockam_command/src/project_member/list.rs]: That's some nitpicking :-) but this doesn't need an if/then/else:
```
!self.enrollers || 
a.deserialized_key_value_attrs().contains(&format!("{}={}", 
    OCKAM_ROLE_ATTRIBUTE_KEY, OCKAM_ROLE_ATTRIBUTE_ENROLLER_VALUE))                      
```

**Review by etorreborre — DISMISSED** [review]

File: implementations/rust/ockam/ockam_command/src/project_member/delete.rs (hunk 1 above)
Comment (adrianbenavides) [reply implementations/rust/ockam/ockam_command/src/project_member/delete.rs]: Oh I like that, I wasn't aware of this!

File: implementations/rust/ockam/ockam_command/src/project_member/list.rs (hunk 2 above)
Comment (adrianbenavides) [reply implementations/rust/ockam/ockam_command/src/project_member/list.rs]: Good catch, fixed too!

**Review by etorreborre — APPROVED** [review]

**Pull request merged**